.github/workflows/linting.yml

```diff
@@ -1,28 +1,20 @@
 ---
 name: Linting
 
-# yamllint disable-line rule:truthy
 on:
   push:
     branches:
       - main
       - renovate/**
   pull_request:
-    types:
-      - opened
-      - ready_for_review
-      - reopened
-      - synchronize
 
 jobs:
   linting:
-    if: github.event.pull_request.draft == false
     runs-on: ubuntu-latest
     steps:
       - name: Checkout source
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
 
-      - name: Run pre-commit over the repo
```

Comment:
just wondering why you removed the name?

Reply:
Purely consistency, the action name should be verbose enough!